docs/public/installation/zonemaster-ldns.md

```diff
@@ -0,0 +1,61 @@
+# Installation: Zonemaster-LDNS
+
+## Recommended installation
+
+The recommended way to install Zonemaster::LDNS is to follow the
+[installation instructions for Zonemaster::Engine] where you will find all
+prerequisites and dependencies for Zonemaster::LDNS before installing it.
+
+
+## Docker
+
+Zonemaster-CLI is available on [Docker Hub], and can be conveniently downloaded
+and run without any installation. See [USING] Zonemaster-CLI for how to run
+Zonemaster-CLI on Docker.
```

Comment:
I don't think so. However, this is present in the original document (zonemaster/zonemaster-ldns/README.md). For now I think it would make the PR lighter to keep the focus on the document structure instead of each document content. Then in a future work, we could rework the content.